### PR TITLE
Allow removing yourself from an org

### PR DESCRIPTION
**Pull request by tsweeney-dust**
I can **totally** see y'all not being into this, if so I'll figure something else out. Very open to feedback here.

We're working on a custom theme to add orgs to the profile page, and there's not a way right now to leave an org via the API if you don't have control over that org.

### CODE REVIEW

**Comment by tsweeney-dust** [comment]
For now I'm going to work around this by calling a 3rd party API which our internal AuthZ is hooked into, and having it enforce our own internal authz and removing you. This isn't a critical need for us.

File: src/main/java/io/phasetwo/service/resource/MembersResource.java [hunk 1]
```diff
@@ -112,4 +112,12 @@ private void canManage() {
               auth.getUser().getId(), organization.getName()));
     }
   }
+
+  private void canDelete(String userId) {
+    var seppuku = (userId == getUser().getId());
```
Comment (xgp) [inline src/main/java/io/phasetwo/service/resource/MembersResource.java:117]: Nits. I'd prefer:
1. we don't use java `var` even though we switched to java 11
2. we use a variable name that doesn't invoke suicide
I know we don't have a style guide, but thanks for changing these.

**Comment by xgp** [comment]
> We're working on a custom theme to add orgs to the profile page

FYI we've created an `admin-portal` app that combines profile and organization management. It will be the same license as our other extensions. Should be released within ~2 weeks.

File: src/main/java/io/phasetwo/service/resource/MembersResource.java (hunk 1 above)
Comment (tsweeney-dust) [reply src/main/java/io/phasetwo/service/resource/MembersResource.java:117]: 1. Sounds good, not a Java dev, just winging this
2. I'm very sorry about that, didn't mean to commit it w/ that name. Was deep in hacking through Keycloak Themes / Java and dumped stress into variable names.

Comment (xgp) [reply src/main/java/io/phasetwo/service/resource/MembersResource.java:117]: No problem. Thanks for making the changes.

**Review by xgp — APPROVED** [review]

File: src/main/java/io/phasetwo/service/resource/MembersResource.java (hunk 1 above)
Comment (tsweeney-dust) [reply src/main/java/io/phasetwo/service/resource/MembersResource.java]: I just dropped the var entirely and put the expression in the if block. LMK if you still want it as a semantically named boolean (like `isRemovingSelf`) or if you want the function named `canRemove` instead

**Comment by xgp** [comment]
This will likely be when we release our `21.0.1` docker image. Within 3-5 days. 

If you need a workaround in the meantime, you can create your own image and replace this jar in the `providers/` dir.

**Pull request merged**